diab_labs/models.py

```
@@ -0,0 +1,15 @@
+
+
+class LabResult(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    patient = models.ForeignKey('patients_core.Patient', on_delete=models.CASCADE)
+    encounter = models.ForeignKey('diab_encounters.Encounter', null=True, blank=True, on_delete=models.SET_NULL)
+    loinc = models.CharField(max_length=40)
+    value = models.DecimalField(max_digits=10, decimal_places=4)
+    unit = models.CharField(max_length=16)
+    taken_at = models.DateTimeField()
+
+    def __str__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)